docs/user_guide.md

```diff
@@ -183,28 +189,29 @@ For a full description of the fields, please refer to the [Agent Network HOCON F
 
 ## LLM configuration
 
-The `llm_config` section of the agent network configuration file specifies which LLM to use
-for the agents in this file. It can be specified:
+The `llm_config` section in the agent network configuration file defines which LLM should be used by the agents.
 
-* at the agent network level, to apply to all agents in this file
-* at the agent level, to apply to a specific agent
+You can specify it at two levels:
+* **Network-level**: Applies to all agents in the file.
+* **Agent-level**: Overrides the network-level configuration for a specific agent.
 
-For a full description of the fields, please refer to the [LLM config](
-    https://github.com/cognizant-ai-lab/neuro-san/blob/main/docs/agent_hocon_reference.md#llm_config) documentation.
+Neuro-SAN includes several predefined LLM providers and models. To use one of these, set the `model_name` key to
+the name of the model you want. A full list of available models can be found in the
+[default LLM info file](https://github.com/cognizant-ai-lab/neuro-san/blob/main/neuro_san/internals/run_context/langchain/llms/default_llm_info.hocon).
 
-For instructions about how to extend the default LLM descriptions shipped with the `neuro-san` library,
-please refer to the [LLM Info HOCON File Reference](
-    https://github.com/cognizant-ai-lab/neuro-san/blob/main/docs/llm_info_hocon_reference.md) documentation.
+> ⚠️ Different providers may require unique configurations or environment variables.
+
+The following sections provide details for each supported provider, including required parameters and setup instructions.
 
 ### OpenAI
 
 To use an OpenAI LLM, set the `OPENAI_API_KEY` environment variable to your OpenAI API key
 and specify which model to use in the `model_name` field:
 
 ```hocon
-    "llm_config": {
-        "model_name": "gpt-4o",
-    },
+"llm_config": {
+    "model_name": "gpt-4o",
+}
```

Comment:
If the linter was not complaining, it might be better to leave the spaces in, as it's less work to copy/paste.